easybuild/easyconfigs/m/MyCC/MyCC-2017-03-01-intel-2019b-Python-2.7.16.eb

```
@@ -0,0 +1,74 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = "CmdCp"
+
+name = "MyCC"
+version = "2017-03-01"
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = "http://docutils.sourceforge.net/"
+description = """Docutils is an open-source text processing system for processing plaintext
+documentation into useful formats, such as HTML, LaTeX, man-pages, open-document or XML.
+It includes reStructuredText, the easy to read, easy to use, what-you-see-is-what-you-get
+plaintext markup language."""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+
+source_urls = [('https://sourceforge.net/projects/sb2nhri/files/MyCC/', 'download')]
+sources = [{'download_filename': '%s-%s.tar.gz' % (name, version), 'filename': 'Tools.tar.gz'}]
```

@Darkless012 You have things reversed here:

```suggestion
sources = [{'download_filename': 'Tools.tar.gz', 'filename': '%s-%s.tar.gz' % (name, version)}]